docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -409,6 +409,24 @@ The following table lists the measurement data that the HRM LED green sensor pro
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
 | values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
 
+<a name="hrm_green_batch"></a>
+## Heart Rate Monitor LED Green batch Sensor
+
+The Heart Rate Monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor must be power-efficient and support batch processing.
```

Comment:
Doubt?
What is the is difference between 
Heart Rate Monitor LED Green batch Sensor and Heart Rate Monitor LED Green Sensor?

The description is the same. 
Can you please verify?

Reply:
These have almost similar purposes, the biggest difference being that they operate differently. 
In addition, the batch-type sensor has more values ​​at the request of MCD

Reply:
We need to add more to the description, such that the descriptions differ.